src/Chapter1.hs

```diff
@@ -508,7 +521,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = if ((abs x) <= (abs y)) then x else y
```

Comment:
`()` are redundant here. I know it might be difficult to read code without them at first, especially when coming from other languages. But having less noise reads to cleaner code :slightly_smiling_face: 

```suggestion
closestToZero x y = if abs x <= abs y then x else y
```

Reply:
thanks! is there cases where they are mandatory? (apart from negative numbers)

Reply:
@marcocapano You mostly use `()` to group arguments for functions or to specify priority when mixing a lot of very different non-standard operators. Also, sometimes in type signatures you use `()` to group types and type variables